NEWS.md

```diff
@@ -110,43 +144,6 @@ New architectures
   * [Power](https://github.com/JuliaLang/julia/issues?utf8=%E2%9C%93&q=label%3Apower):
     [#16455], [#16404]
 
-Breaking changes
```

Comment:
I would have preferred all the breaking things and deprecations to be together. The flow that makes sense to me is - new tools stuff, new language things, new compiler things, library improvements, breaking changes and deprecations.

I can see that perhaps breaking changes and language changes need to be clubbed together - but as I look deeper, there seem to be things in language changes that ought to be in breaking (like case-sensitive using and import).

How can we simplify the structure and the communication in this file?